www/include/reporting/dashboard/ajaxReporting_js.php

```diff
@@ -34,18 +34,22 @@
  */
     
 require_once realpath(dirname(__FILE__) . "/../../../../config/centreon.config.php");
+require_once './class/centreonEscaping.class.php';
 
 if ($type == "Service") {
-    $arg = "id=".$service_id."&host_id=".$host_id;
+    $arg = "id=".Esc::forUrlValue($service_id)."&host_id=".Esc::forUrlValue($host_id);
 } else {
-    $arg = "id=".$id;
+    $arg = "id=".Esc::forUrlValue($id);
 }
 
-$arg .= "&color[UP]=".$oreon->optGen["color_up"]."&color[UNDETERMINED]=".$oreon->optGen["color_undetermined"].
-        "&color[DOWN]=".$oreon->optGen["color_down"]."&color[UNREACHABLE]=".$oreon->optGen["color_unreachable"].
-        "&color[OK]=".$oreon->optGen["color_ok"]."&color[WARNING]=".$oreon->optGen["color_warning"].
-        "&color[CRITICAL]=".$oreon->optGen["color_critical"]."&color[UNKNOWN]=".$oreon->optGen["color_unknown"];
-$arg = str_replace("#", "%23", $arg);
+$arg .= "&color[UP]=".Esc::forUrlValue($oreon->optGen["color_up"]).
```

Comment:
PSR2 -> please add a space before and after the dots
AND please split this line (120 chars max per line)

Reply:
done